ui/src/services/clutter-dna.ts

```diff
@@ -30,7 +30,7 @@ export const callZome = async <T>(
   payload: CallZomeRequest["payload"]
 ) => {
   const result: { type: "ok"; data: T } = await useClientStore().callZome({
-    roleId: CLUTTER_ROLE_ID,
+    roleId: CLUTTER_ROLE_NAME,
```

Comment:
should be roleName here?

Reply:
the ui-common-library doesn't support that yet